packages/joint-core/types/joint.d.ts

```diff
@@ -263,6 +263,10 @@ export namespace dia {
 
         removeCells(cells: Cell[], opt?: Cell.DisconnectableOptions): this;
 
+        transferCellEmbeds(sourceCell: Cell, targetCell: Cell, opt?: { [key: string]: any }): void;
+
+        transferCellConnectedLinks(sourceCell: Cell, targetCell: Cell, opt?: { [key: string]: any }): void;
```

Comment:
The `opt` here is a `Graph.ConnectionOptions`. You can for instance, reconnect only the `inbound` links.